src/AopCode.php

```
@@ -139,7 +139,7 @@
 
             $isClassSignatureEnds = $inClass && $text === '{';
             if ($isClassSignatureEnds) {
-                $this->addIntercepterTrait();
+                $this->addInterceptorTrait($sourceClass);
```

PHPStan (2.0.3)

Method Ray\Aop\AopCode::addInterceptorTrait() invoked with 1 parameter, 0 required.


(arguments.count)